docs/user_guide/runtime_ima.rst

```diff
@@ -110,111 +111,202 @@ This will then result in `[filename]` being available for Agent provisioning.
     the runtime policy hash to ensure no tampering occurs when transferring to other
     machines.
 
-Alongside building a runtime policy from `initramfs`, you could also generate good
-hashes for your applications files or admin scripts that will run on the
-remotely attested machine.
 
-Excludes List
-~~~~~~~~~~~~~
+Creating more Complex Policies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The second script allows the user to build more complex policies by providing options to include:
+keyring verification, IMA verification keys, generating allowlist from IMA measurement log
+and extending existing policies. The :code:`create_policy` script is `available here <https://github.com/keylime/keylime/blob/master/scripts/create_policy>`_.
```

Comment:
At some point we can maybe promote it to an official tool... Ideally after some feedback and also with support for device manager policies (?).

Reply:
Yes, I think that is a good idea. Especially because we are deprecating passing IMA verification keys directly to the tenant. We can probably merge the create_mb_refstate script also into that tool.